etl/transform.py

```diff
@@ -111,9 +127,9 @@ def parse_args():
     dataset_id = args.dataset_id
 
     df = (
-        spark.read.options(delimiter="\t", header=False)
+        spark.read
         .schema(RawDataset.schema)
-        .csv(source)
+        .parquet(source)
```

Comment:
Here we could reduce I/O by filtering `source` by `snapshot`.

Maybe something we could look into as part of T277557?

Reply:
sounds good to me!